ui/component/or-asset-tree/src/index.ts

```diff
@@ -305,8 +314,11 @@ export class OrAssetTree extends subscribe(manager)(LitElement) {
         return false;
     }
 
-    protected render() {
+    protected isFilteringAndNotMatching(assetId: string | undefined): boolean {
+        return assetId !== undefined && this._filterValue.length > 0 && this._foundAssets.length > 0 && this._foundAssets.indexOf(assetId) === -1;
+    }
```

Comment:
Is this function necessary? Can this logic be stored as state in the `UiAssetTreeNode` when you iterate through the tree to apply filtering?

Reply:
Indeed, I include necessary info now in the recursive function and avoid the usage of an additional attribute in the class :-)